keras/utils/generic_utils.py

```diff
@@ -5,9 +5,9 @@
 import six
 
 
-def get_from_module(identifier, module_params, module_name, instantiate=False, kwargs=None):
+def get_from_module(identifier, module_params, module_name, instantiate=False, kwargs=None, custom_layers={}):
```

Comment:
You should not be modifying `get_from_module`. Instead you should be merging the `custom_layers` dict with the layer dict from `globals`.